<launch>
    <group ns="/decawave">
        <node pkg="ros_decawave" type="decawave_driver.py" name="decawave_driver" output="screen">
	    <rosparam command="load" file="$(find ros_decawave)/config/parameters.yaml" />
            <!--param name="port" value="/dev/serial/by-id/usb-SEGGER_J-Link_000760038890-if00"/-->
            <param name="baudrate" value="115200"/>
            <param name="tf_publisher" value="True"/>
            <param name="tf_reference" value="world"/>
            <!--param name="tag_name" value="DWC4B8"/-->
            <!--param name="rate" value="10"/-->
        </node>

        <!-- Transform tf to path -->
        <!--node pkg="hector_trajectory_server" type="hector_trajectory_server" name="tag_trajectory" output="screen">
            <param name="target_frame_name" type="string" value="world" />
            <param name="source_frame_name" type="string" value="t_DWC0AF" />
            <param name="trajectory_update_rate" type="double" value="10" />
            <param name="trajectory_publish_rate" type="double" value="10" />
            <remap from="/trajectory" to="/trajectory_DWC0AF"/>
        </node-->
    </group>

    
    <!--group ns="/dwc4b8">
        <node pkg="ros_decawave" type="decawave_driver.py" name="decawave_driver" output="screen">
            <param name="port" value="/dev/serial/by-id/usb-SEGGER_J-Link_000760038890-if00"/>
            <param name="baudrate" value="115200"/>
            <param name="tf_publisher" value="True"/>
            <param name="tf_reference" value="world"/>
            <param name="tag_name" value="t_DWC4B8"/>
            <param name="rate" value="10"/>
        </node>

        <node pkg="hector_trajectory_server" type="hector_trajectory_server" name="tag_trajectory" output="screen">
            <param name="target_frame_name" type="string" value="world" />
            <param name="source_frame_name" type="string" value="t_DWC4B8" />
            <param name="trajectory_update_rate" type="double" value="10" />
            <param name="trajectory_publish_rate" type="double" value="10" />
            <remap from="/trajectory" to="/trajectory_DWC4B8"/>
        </node>
    </group-->

    <!-- Show in Rviz   -->
    <!--node pkg="rviz" type="rviz" name="rviz" args="-d $(find ros_decawave)/config/config.rviz"/-->

</launch>
